faq.md

```diff
@@ -31,10 +31,16 @@ _Open-source software steward_ is a term defined in [Article 3(14)][] of the CRA
 
 > ‘open-source software steward’ means a legal person, other than a manufacturer, that has the purpose or objective of systematically providing support on a sustained basis for the development of specific products with digital elements, qualifying as free and open-source software and intended for commercial activities, and that ensures the viability of those products;
 
+_Open-source software steward_ are subject to a "light-touch and tailor-made regulatory regime" ([Recital 19][]), defined in [Article 24][].
+
 #### Q: Can an solo maintainer be considered to be an _open-source software steward_?
 
+No. As defined in [Article 3(14)][], an _open-source software steward_ must be a _legal person_ (e.g. a company, an organization, etc.) in contrast with a _natural person_ (i.e. a human being). The obligations of _open-source software stewards_ described in [Article 24][] therefore do not apply to solo maintainers. It is worth noting however, that _natural persons_ are subject to the same obligations as _legal persons_ would be should they monetize their poject.
+
 #### Q: Can a loosely organized group of maintainers be considered to be an _open-source software steward_?
 
+In [Article 3(14)][], the CRA defines _open-source software steward_ as a "legal person, other than a manufacturer." In European regulation, groups of people acting in concert can be considered as a legal person, whether or not they have formalized their relationship by creating a society, trust, foundation, or company. So it is possible to that in some cases, a group of open source developers maintaining an open source project together might be considered to be an _open-source software steward_, even if it is only loosely organized together.
```

Comment:
I (literally) don’t know if this is true for all EU countries. I’ve seen this argued for some countries, and I’m inclined to believe those who do, but as currently written it seems to convey that this is _true for all of the EU_, more specifically through “European regulation”, ie. harmonized.

This is outside my area of expertise, I’m just flagging my understanding of what’s currently written here and noting that I am unable to give a thumbs up. Illu, do you know?